### DISCUSSION TITLE
NocoDB with Oracle [May 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5676] [Discord Threads]
> 🙋‍♂️ **Keyruu**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@137900351289425920> can you help us understand more about your usage scenario?  
#  
> 🙋‍♂️ **Keyruu**  
#### We need a UI for editing data inside of an Oracle DB  
#  
> 💁 **o1lab**  
#### currently, there are issues with oracle lib and bundling them to our docker .. otherwise we've a branch where it works in dev mode  
#  
> 🙋‍♂️ **Keyruu**  
#### oh okay, ill try that then thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1095669045799432294)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by liudonghua123** [comment]
Any updates? The latest https://github.com/oracle/node-oracledb has thin mode which does not need to install Oracle Instant Client.

I am also working on add oracle support.

**Reply by ComeFromPast** [reply]
你好，我目前也在尝试将Oracle数据库添加为数据源，请问您有什么好的可行方法了吗，我目前仍未发现解决办法，我所安装的NocoDB目前并不支持Oracle作为数据源
